apps/ui/src/components/ProposalLabels.vue

```diff
@@ -27,6 +29,15 @@ const validLabels = computed(() => {
       :key="label.id"
       :label="label.name"
       :color="label.color"
+      :to="
+        withLink
+          ? {
+              name: 'space-proposals',
+              params: { space: `${space.network}:${space.id}` },
+              query: { 'labels[]': label.id }
```

Comment:
we can use `label` instead
<img width="303" alt="Untitled 7" src="https://github.com/user-attachments/assets/7eae9f54-788a-48a3-95a3-4f24348dd329">